components/core/tests/test-sql.cpp

```diff
@@ -0,0 +1,23 @@
+#include <sstream>
```

Comment:
```suggestion
#include <memory>
#include <sstream>
```
For `std::dynamic_pointer_cast`